lib/class-wp-theme-json-gutenberg.php

```diff
@@ -144,7 +144,7 @@ class WP_Theme_JSON_Gutenberg {
 			'path'              => array( 'color', 'duotone' ),
 			'prevent_override'  => array( 'color', 'defaultDuotone' ),
 			'use_default_names' => false,
-			'value_func'        => 'gutenberg_get_duotone_filter_property',
+			'value_func'        => null, // Don't output CSS Custom Properties for duotone.
 			'css_vars'          => '--wp--preset--duotone--$slug',
 			'classes'           => array(),
 			'properties'        => array( 'filter' ),
```

Comment:
I'm pretty sure the entire duotone section of `PRESETS_METADATA` can be removed since generating the preset is done by duotone.php now.

Reply:
Actually, it looks like the constructor still uses it for keying presets by origin which we need later. That being said, I still think we can remove the `css_vars` key and have that solely defined in duotone.php as part of the cleanup.

Also, I noticed that the theme.json `styles.blocks['core/block'].filter.duotone` filters are being applied on the frontend, but not the editor, so there are also likely changes that need to be done in the block-editor duotone.js hook and use-global-styles-output.js as well.